### PR TITLE
Latvian localizations

### PR DESCRIPTION
**Pull request by Exerra**
This pull request adds i18n localizations for Latvian. Most of it is done, but I do have some questions before marking this PR as ready.

1. In the `gen.php` file, where are the `last_*` (starting from line 94) locales used? I need more info on this to be able to figure out the best declension.
2. In the `gen.php` file, what is the `dir` locale (line 122) used for?
3. In the `gen.php` file, where is the `category_empty` (line 128) locale used? I already translated, and it should be fine as is, but I want to double-check.
4. In the `conf.php` file, in line 133, what is `n°` meant to be?
5. I noticed that in `gen.php` the `lang` locale (line 140) contains names for each language. Do I need to go ahead and paste the LV lang locale in every single languages locale files?

### CODE REVIEW

**Comment by Alkarex** [comment]
Hello and welcome!
Let us know if you need assistance with anything. Besides a few syntax errors (see automated tests), there is also a need to add an entry about the new language in the other languages (there is a command line to help with that - I cannot explain now, but maybe @aledeg or @Frenzie could provide a more precise example)

**Comment by Frenzie** [comment]
I don't have the repo cloned on this new PC atm but cli/manupulate.translation.php should be fairly self-explanatory when you run it.

File: app/i18n/lv/gen.php [hunk 1]
```diff
@@ -73,44 +73,44 @@
...
 		'fri' => 'Pk',
-		'jan' => 'Jan.',
+		'jan' => 'Jan.',	// IGNORE
 		'january' => 'Janvāris',
 		'jul' => 'Jūlijs',
 		'july' => 'Jūlijs',
 		'jun' => 'Jūnijs',
 		'june' => 'Jūnijs',
-		'last_2_year' => 'Pēdējie divi gadi', // DIRTY: Not sure where this is used. Depending on the usage might need a different declension. Same goes for every last_* translation
+		'last_2_year' => 'Pēdējie divi gadi',
```
Comment (Alkarex) [inline app/i18n/lv/gen.php:94]: This is used in the page for the statistics
https://github.com/FreshRSS/FreshRSS/blob/1ee2a3d72d97df34b5531ae6787f996ac8d62a2b/app/Controllers/statsController.php#L99-L126

**Comment by Alkarex** [comment]
I have fixed the various technicalities https://github.com/FreshRSS/FreshRSS/pull/5254/commits/37d8ed983f6ab83dbc7e2b52fe3da51e4df2c36c

File: app/i18n/lv/gen.php [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+		'sep' => 'Sept.',	// IGNORE
+		'september' => 'Septembris',
+		'sun' => 'Sv',
+		'thu' => 'C',
+		'today' => 'Šodien',
+		'tue' => 'O',
+		'wed' => 'T',
+		'yesterday' => 'Vakar',
+	),
+	'dir' => 'ltr',	// IGNORE
```
Comment (Alkarex) [inline app/i18n/lv/gen.php:122]: Left-to-right language

File: app/i18n/lv/gen.php [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+	'freshrss' => array(
+		'_' => 'FreshRSS',	// TODO
+		'about' => 'Par FreshRSS',
+	),
+	'js' => array(
+		'category_empty' => 'Iztukšot kategoriju',	// DIRTY
```
Comment (Alkarex) [inline app/i18n/lv/gen.php]: Used when a category does not contain any feed, in the feed management page

File: app/i18n/lv/conf.php [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+			'tags' => 'Rādīt pēc birkas',
+			'type' => 'Veids',
+		),
+		'get_all' => 'Rādīt visus rakstus',
+		'get_category' => 'Rādīt kategoriju “%s”',
+		'get_favorite' => 'Rādīt mīļākos rakstus',
+		'get_feed' => 'Rādīt barotni “%s”',
+		'name' => 'Vārds',
+		'no_filter' => 'Bez filtra',
+		'number' => 'Pieprasījums n°%d',	// DIRTY
```
Comment (Alkarex) [inline app/i18n/lv/conf.php]: *n°* is abbreviation of *number*

**Comment by Exerra** [comment]
Alright, thank you for the replies and fixes! Fixed 2 spots, so everything should be fine now. I will mark this PR as ready for review.

**Comment by Alkarex** [comment]
Sounds good. Please remove the remaining `// DIRTY` annotations (which are used for strings, which have been partially translated)

**Comment by Exerra** [comment]
All of the `// DIRTY` comments should be removed now

**Comment by Alkarex** [comment]
Thanks! Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/edge/CREDITS.md

**Pull request merged**